dts/bindings/charger/ti,bq24190.yaml

```
@@ -1,7 +1,21 @@
 # Copyright 2023 Cirrus Logic, Inc.
+# Copyright 2025 Palta Tech, S.A
 # SPDX-License-Identifier: Apache-2.0
 
-description: Texas Instruments family of BQ24190 of charging ICs
+description: |
+  BQ24190 I2C controlled, 1-Cell, 4.5-A, USB/Adapter Charger with Narrow VDC Power Path Management
+  and USB OTG
```

I would recommend keeping the initial (first sentence) description short. Also wonder if that compatible shouldn't have been named `ti,bq2419x`...

```suggestion
description: |
  BQ24190 family of charging ICs.

  BQ2419x are I2C controlled, 1-Cell, 4.5-A, USB/Adapter Chargers with Narrow VDC Power Path
  Management and USB OTG
```

---

I have modified according to your suggestions @kartben  and also think that would be appropriate to define `bq2419x` instead of `bq24190` only. However, this brings some additional changes that was not expected on this PR. 

Please let me know, if this is the right path to follow. I have seen that its possible to make changes on dts folder but also in between drivers folders.  

@rriveramcrus, when trying to rename `bq24190`, to `bq2419x`, it seems git take this as a new file with all modifications. Is this okey on your end?

---

> Oh sorry I should have been clear that I was by no means suggesting or asking you that it should be changed in this PR! It's great that you started looking into it though! It would be great to keep this for a separate PR indeed -- plus, this will be a breaking change for users so this is something to take into consideration as to whether it's a worthwhile change.

This is fine and thanks for clarify this. I will make a separate PR and see if this is indeed, required.

---

> but fwiw if you use `git mv` git should see this as a proper rename
> 
> ```
> git mv dts/bindings/charger/ti,bq24190.yaml dts/bindings/charger/ti,bq2419x.yaml 
> ```

This is awesome ! Did not know that you can do that. Thanks.